specification/monitor/resource-manager/Microsoft.Monitor/preview/2021-06-03-preview/examples/MonitoringAccountsCreate.json

```diff
@@ -0,0 +1,68 @@
+{
+  "parameters": {
+    "subscriptionId": "703362b3-f278-4e4b-9179-c76eaf41ffc2",
+    "resourceGroupName": "myResourceGroup",
+    "monitoringAccountName": "myMonitoringAccount",
+    "api-version": "2021-06-03-preview",
+    "monitoringAccountProperties": {
+      "location": "eastus"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/703362b3-f278-4e4b-9179-c76eaf41ffc2/resourceGroups/myResourceGroup/providers/Microsoft.Monitor/accounts/myMonitoringAccount",
+        "name": "myMonitoringAccount",
+        "type": "Microsoft.Monitor/accounts",
+        "location": "eastus",
+        "tags": {
+          "tag1": "A",
+          "tag2": "B"
+        },
+        "properties": {
+          "accountId": "2df515bf-c3ce-4920-84d4-1d9d16542d9f",
+          "metrics": {
+            "prometheusQueryEndpoint": "https://myMonitoringAccount-v8hx.eastus.prometheus.monitor.azure.com",
+            "internalId": "mac_2df515bf-c3ce-4920-84d4-1d9d16542d9f"
+          },
+          "provisioningState": "Succeeded",
+          "defaultIngestionSettings": {
+            "dataCollectionRule": "/subscriptions/703362b3-f278-4e4b-9179-c76eaf41ffc2/resourceGroups/MA_myMonitoringAccount_eastus_managed/providers/Microsoft.Insights/dataCollectionRules/myMonitoringAccount",
```

Comment:
>subscriptions [](http://example.com/codeflow?start=36&length=13)

Doesnt the user need to specify these properties ? are they created implicitly ?

Reply:
Yes, this is created by the RP automatically, in a managed Resource Group